### DISCUSSION TITLE
Support for ignoring forks

### DISCUSSION THREAD

**Discussion by martincostello** [lindell/multi-gitter#340] [Ideas]
When running multi-gitter against my personal GitHub account, it finds various forks I have of other people's/organization's repositories.

I'd like to be easily able to apply a change to all repositories that I own which are not forks without having to manually curate a list of repositories.

For example, using a configuration like this:

```
user:
  - martincostello

ignore-forks: true
```

**Comment by lindell** [comment]
Seems like a reasonable option to have 🙂 Although I would call it `skip-forks` to align with the naming of `skip-repo`.

It will not be on my prio to implement, but I would be happy to merge a PR. If you want to take a look, you can use "topics" as a reference that was implemented to to long ago, but this should be way easier. 

https://github.com/lindell/multi-gitter/pull/320/files

**Comment by martincostello** [comment]
Included in [v0.46.0](https://github.com/lindell/multi-gitter/releases/tag/v0.46.0).
